TWLight/users/signals.py

```diff
@@ -106,8 +107,17 @@ def update_existing_bundle_authorizations(sender, instance, **kwargs):
         if previously_available and previously_bundle:
             remove_from_auths = True
 
+        # Case when partner is no more available
+        # but set to Bundle at the same time.
```

Comment:
Let's add the Phab ticket at the beginning of this comment. Also, let's change the wording a bit from `Case when partner is no more available` to `Case when a partner does not have an Available status and has a Bundle authorization method set in the same operation.`

Reply:
Thank you for the suggestions :) I have updated the comment